### PR TITLE
Add ability to bypass the domain filters

### PR DESCRIPTION
**Pull request by itsthejoker**
This is a three-part update and will need to be handled with care to make sure all three parts are verified before deploying.

* u/ToR PR: https://github.com/TranscribersOfReddit/TranscribersOfReddit/pull/53
* tor-core PR: https://github.com/TranscribersOfReddit/tor_core/pull/8
* wiki update: https://www.reddit.com/r/TranscribersOfReddit/wiki/format/other and https://www.reddit.com/r/TranscribersOfReddit/wiki/subreddits/domain-filter-bypass

What this PR does is enable adding subreddits to the `domain-filter-bypass` wiki page, allowing all content posted there to be crossposted to r/ToR. This option will currently be used for two subreddits: r/DescriptionPlease and r/Futurology.

### CODE REVIEW

File: tor_core/initialize.py [hunk 1]
```diff
@@ -106,10 +106,12 @@ def populate_formatting(config):
     config.audio_formatting = ''
     config.video_formatting = ''
     config.image_formatting = ''
+    config.other_formatting = ''
 
     config.audio_formatting = get_wiki_page('format/audio', config)
     config.video_formatting = get_wiki_page('format/video', config)
     config.image_formatting = get_wiki_page('format/images', config)
+    config.image_formatting = get_wiki_page('format/other', config)
```
Comment (arfie) [inline tor_core/initialize.py]: Should probably be `other_formatting` here :P

File: tor_core/initialize.py [hunk 2]
```diff
@@ -184,6 +186,18 @@ def populate_subreddit_lists(config):
         )
     )
 
+    config.subreddits_domain_filter_bypass = get_wiki_page(
+        'subreddits/domain-filter-bypass', config
+    ).split('\r\n')
```
Comment (arfie) [inline tor_core/initialize.py:191]: Not really a problem as it is now, but you can also use `splitlines` which will work regardless of newline type.

File: tor_core/initialize.py (hunk 1 above)
Comment (captcoe) [reply tor_core/initialize.py]: Nah, the folder above is called `format`, it should be pretty obvious

Comment (captcoe) [reply tor_core/initialize.py]: Woops nevermind

Comment (itsthejoker) [reply tor_core/initialize.py]: Great catch!

**Review by TheLonelyGhost — APPROVED** [review]
Oops! TIL that being a code owner blocks merges unless I review them.

Looks good for now. I agree with Ruud's review in that `splitlines` would be better, but a non-issue for right now.

**Pull request merged**